src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentMonetaryAccount.java

```diff
@@ -31,7 +31,6 @@
   /**
    * Object type.
    */
-  private static final String OBJECT_TYPE = "AttachmentMonetaryAccount";
```

Comment:
Is it intended that the comment is still there when there are no constants to which the comment belongs to?

Reply:
Ah no, the constant generator is not checking to see if it actually has constants to put underneath the doc block :sweat_smile:. Nice catch!

Reply:
Fixed in generator, pending review.